CONTRIBUTING.md

```diff
@@ -35,6 +35,8 @@ All committers should feel responsible to read incoming issues. When you read a
 * If you are not familiar enough with the topic of the issue, you might find someone who is and delegate the action.
 
 ## Set up your Eclipse Workspace
+Before starting the setup process, make sure that you have both an [Eclipse](https://accounts.eclipse.org/user/register) and and a [GitHub](https://github.com/join) account. Also make sure that you have _disabled_ two factor authorisation on GitHub during the setup process, as the installer currently cannot handle that.
```

Comment:
Please add a link to where to find details why this is currently not possible (bug, forum entry. etc.)

Add link to https://github.com/settings/security at "two factor authorisation".